dev/PowerNotifications/PowerNotifications.idl

```diff
@@ -0,0 +1,115 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+// REVIEW: We're not doing doing any versioning within IDL/WinMDs, right?
+
+namespace Microsoft.ProjectReunion
+{
+    // Enums duplicated from IPowerManagerStatics
+    enum EnergySaverStatus
+    {
+        Disabled = 0,
+        Off,
+        On
+    };
+
+    enum BatteryStatus
+    {
+        Uninitialized = -1,
```

Comment:
Don't define an `Uninitialized` value, and WinRT enums really don't like `-1` as a value.

Reply:
What would be a better approach then?

Reply:
Just remove this line entirely and set the first (`NotPresent`) value to `0` and use that as the "unspecified" value.

Reply:
You can still privately use `-1` to mean Uninitialized, but it doesn't need to be part of the public API surface.